src/Microsoft.Framework.PackageManager/Algorithm/DFPostOrderWalker.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using System.Collections.Generic;
+
+namespace Microsoft.Framework.PackageManager.Algorithm
+{
+    /// <summary>
+    /// Walk the graph in depth-first post order 
+    /// </summary>
+    public class DFPostOrderWalker<T, K>
```

Comment:
Should the file name be DepthFirstPostOrderWalker.cs instead of DFPostOrderWalker.cs ?